recipe/meta.yaml

```diff
@@ -1,35 +1,48 @@
-{% set version = "1.10" %}
-{% set sha256 = "bcbe8ada8c435a253b1fa438f505dd311832a88f52316fd7d30342b106a4242c" %}
-{% set runc_version = "0.1.0" %}
-{% set runc_sha256 = "78ae42905c0f26fcfa9353e48e1a001b5b5c3c94b33e2ceacc6bc84c23c61a88" %}
+{% set goname = "github.com/tianon/gosu" %}
+{% set version = "1.11" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+# The runc version much match the upstream Dockerfile
+# https://github.com/tianon/gosu/blob/master/Dockerfile
+{% set runc_goname = "github.com/opencontainers/runc" %}
+{% set runc_version = "1.0.0-rc5" %}
+{% set runc_pkg_src = ('src/'+runc_goname).replace("/",os.sep) %}
 
 package:
-  name: gosu
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  - folder: src/github.com/tianon/gosu
-    url: https://github.com/tianon/gosu/archive/{{ version }}.tar.gz
-    sha256: {{ sha256 }}
+  - folder: {{ pkg_src }}
+    url: https://{{ goname }}/archive/{{ version }}.tar.gz
+    sha256: d03208fd7ef983e4a606f861a5efcdc5a1980952a91febba3f67057800da6701
 
-  - folder: src/github.com/opencontainers/runc
-    url: https://github.com/opencontainers/runc/archive/v{{ runc_version }}.tar.gz
-    sha256: {{ runc_sha256 }}
+  - folder: {{ runc_pkg_src }}
+    url: https://{{ runc_goname }}/archive/v{{ runc_version }}.tar.gz
+    sha256: 770e180d1cca1f3c1a67512ebf9720caafe97af4fc2fe12fa43a3cf7bfa77522
 
 build:
-  number: 1001
+  number: 1000
   skip: true  # [not linux]
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v -ldflags '-d -s -w'
+    - go test -v
```

Comment:
Unfortunately, we need the entire source code if we want to test. This is similar to our calls to `make check` for c/c++ programs.